src/pages/Defi/components/OpportunityCardList.tsx

```diff
@@ -43,11 +40,13 @@ export const VaultList = ({ balances }: { balances: UseEarnBalancesReturn }) =>
         gridTemplateColumns={{ base: 'repeat(1, 1fr)', lg: 'repeat(3, 1fr)' }}
         gridGap={6}
       >
-        {activeVaults.map(vault => {
-          return <StakingCard isLoaded={true} key={vault.vaultAddress} {...vault} />
+        {activeOpportunities.map(opportunity => {
```

Comment:
I believe a filter() will always return an array even if there are no matching items. So I don't think this would be an issue. The map would just map over nothing.